modules/apps/transfer/types/msgs_test.go

```diff
@@ -129,17 +132,19 @@ func TestMsgUpdateParamsGetSigners(t *testing.T) {
 
 	for _, tc := range testCases {
 		tc := tc
-
-		msg := types.MsgUpdateParams{
-			Signer: tc.address.String(),
-			Params: types.DefaultParams(),
-		}
-		if tc.expPass {
-			require.Equal(t, []sdk.AccAddress{tc.address}, msg.GetSigners())
-		} else {
-			require.Panics(t, func() {
-				msg.GetSigners()
-			})
-		}
+		suite.Run(tc.name, func() {
+			address := tc.address
+			msg := types.MsgUpdateParams{
+				Signer: tc.address.String(),
+				Params: types.DefaultParams(),
+			}
+			signers, _, err := suite.chainA.GetSimApp().AppCodec().GetMsgV1Signers(&msg)
```

Comment:
this one can also be refactored to use `moduletestutil.MakeTestEncodingConfig` instead of `suite`

Reply:
It good now 🥇